commit 9c4c8cab4de50f98dd50cee2eeef61f204061c10 (refs/changes/17/4817/2)
Author: Jason King <jason.king@joyent.com>
Date:   2018-09-10T20:10:51-05:00 (1 year, 1 month ago)
    
    OS-7200 would like thread name API (fix debug)
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/usr/src/cmd/mdb/common/modules/genunix/thread.c b/usr/src/cmd/mdb/common/modules/genunix/thread.c
index fadbfec863..410f9973d8 100644
--- a/usr/src/cmd/mdb/common/modules/genunix/thread.c
+++ b/usr/src/cmd/mdb/common/modules/genunix/thread.c
@@ -782,7 +782,6 @@ int
 stackinfo(uintptr_t addr, uint_t flags, int argc, const mdb_arg_t *argv)
 {
 	kthread_t t;
-	proc_t p;
 	uint64_t *ptr;  /* pattern pointer */
 	caddr_t	start;	/* kernel stack start */
 	caddr_t end;	/* kernel stack end */
